spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb

```diff
@@ -44,82 +49,99 @@
         let!(:jobseeker) { create(:jobseeker) }
 
         context "when jobseeker is signed in" do
-          before { login_as(jobseeker, scope: :jobseeker) }
+          let(:jobseeker_signed_in?) { true }
 
-          it "renders a link to job alerts dashboard" do
-            click_on I18n.t("subscriptions.confirm.view_job_alerts")
+          it "redirects to job alerts dashboard" do
             expect(current_path).to eq(jobseekers_subscriptions_path)
           end
         end
 
         context "when jobseeker is signed out" do
-          it "renders a link to job alerts dashboard" do
-            click_on I18n.t("subscriptions.confirm.view_job_alerts")
+          it "renders a sign in prompt that redirects to job alerts dashboard" do
+            within "#sign-in-prompt" do
+              expect(page).to have_content(I18n.t("subscriptions.jobseeker_account_prompt.heading.sign_in"))
+              click_on I18n.t("subscriptions.jobseeker_account_prompt.link.sign_in")
+            end
             expect(current_path).to eq(new_jobseeker_session_path)
             sign_in_jobseeker
             expect(current_path).to eq(jobseekers_subscriptions_path)
           end
         end
       end
+
+      context "when jobseeker does not have an account" do
+        it "renders a create account prompt that redirects to job alerts dashboard" do
+          within "#sign-up-prompt" do
+            expect(page).to have_content(I18n.t("subscriptions.jobseeker_account_prompt.heading.sign_up"))
+            click_on I18n.t("subscriptions.jobseeker_account_prompt.link.sign_up")
+          end
+          expect(current_path).to eq(new_jobseeker_registration_path)
+          sign_up_jobseeker
+          visit first_link_from_last_mail
+          expect(current_path).to eq(jobseekers_subscriptions_path)
+        end
+      end
     end
   end
 
-  context "when a location category search is carried out" do
-    let(:location_category_search?) { true }
-    let(:location) { "London" }
-    let!(:location_polygon) { create(:location_polygon, name: "london") }
+  describe "location search" do
+    context "when a location category search is carried out" do
+      let(:location_category_search?) { true }
+      let(:location) { "London" }
+      let!(:location_polygon) { create(:location_polygon, name: "london") }
 
-    scenario "can successfuly subscribe to a new alert" do
-      visit jobs_path
-      and_perform_a_search
-      and_click_job_alert_link
+      scenario "can successfuly subscribe to a new alert" do
```

Comment:
I think that the latest rubocop recommendations are that we change `scenario` with `it`. Not 100% sure on that though

Reply:
That's cool with me anyway